pywinauto/timings.py

```diff
@@ -319,11 +319,45 @@ def WaitUntil(
             
     return func_val
 
+#=========================================================================
+def wait_until(

```

Comment:
Sorry for missunderstanding, it is not clear how this suggestion - "WaitUntil should be renamed to wait_until" could collaborate with this one - "The idea is to keep the implementation in one place but still have the both the decorator and the regular function"?